homeassistant/components/systemmonitor/sensor.py

```diff
@@ -515,6 +505,7 @@ async def async_setup_sensor_registry_updates(
 
     def _update_sensors() -> None:
         """Update sensors and store the result in the registry."""
+        _LOGGER.debug("Updating %s", sensor_registry.keys())
```

Comment:
keys() will get called even if debug logging doesn't happen. You can avoid this with isEnabledFor